Finish mobile

ragangithub opened the pull request:
# In this branch
- added the contact section

zieeco commented on index.html:
```diff
@@ -236,5 +236,42 @@ <h3>About Myself</h3>
+              ></textarea>
+            </li>
+            <li class="button">
+              <button type="submit">Get in Touch</button>
```
- Hey, please, let's kindly improve the design on our button by removing the black border added, add the interaction to the button to indicate the `hover, active, enabled, disabled` state to improve the user experience when they interact with the button.


![Screenshot from 2023-03-28 14-25-13](https://user-images.githubusercontent.com/59397159/228252059-2b377771-fb21-4200-9bf7-d237696dee9e.png)
![Screenshot from 2023-03-28 14-24-26](https://user-images.githubusercontent.com/59397159/228252063-168e9be7-fc52-4753-a169-e28faca642ba.png)

OyePriscilla replied:
- [ ] Kindly implement the button interactions properly according to the design as suggested earlier; when we reproduce a given design perfectly, it shows that we pay attention to details. 👍 

## Figma design
<img width="282" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/228341456-12e8b219-d025-4f8a-a09d-16a5deb8f45e.png">

zieeco reviewed (changes requested):
Hi @Team,


<img height="250" src="https://media0.giphy.com/media/ZubZqIeSsZ60t0ID9l/giphy-downsized-medium.gif"/>



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ragangithub commented:
# In this branch
- added required to input fields
- removed the black border from the button and added interactions to it
- corrected background image

OyePriscilla commented on style.css:
```diff
@@ -292,6 +292,112 @@ section.works {
+  display: flex;
+  flex-direction: column;
+  gap: 0.8rem;
+  background-image: url("/images/contact-form-shapes-mobile.svg");
```
- [ ] The background image is not yet displaying, I observed that you did not reference the source file properly. Perhaps, you can try any of the two options below:
```suggestion
  background-image: url("./images/contact-form-shapes-mobile.svg");
```

```suggestion
  background-image: url("images/contact-form-shapes-mobile.svg");
```

OyePriscilla reviewed (changes requested):
Hi @ragangithub and @OTMANE-ECHCHAFYKY 👍 👍, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425724-8ca5825c-b02a-4983-b976-4bd0a2f70f9a.gif)

 ## Highlights
Create forms with HTML5 validations ✔️ 
Collect data by using the [Formspree service](https://formspree.io/html) ✔️ 
Made some required changes ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [optional] Kindly remove the microverse logo from your readme file or you replace it with yours if you have any. This will make your repo to be more informative. 👍  

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ragangithub commented:
# in this branch
- corrected background image by using the suggestion background-image: URL("./images/contact-form-shapes-mobile.svg");
- updated the button interaction according to Figma design
